javaparser-core/src/main/java/com/github/javaparser/ast/visitor/EqualsVisitor.java

```diff
@@ -147,6 +147,13 @@ public Boolean visit(final CompilationUnit n, final Visitable arg) {
         return true;
     }
 
+    // TODO Implement EqualsVisitor for StubUnit
```

Comment:
That should get auto-generated when you add it to the list of node types in the metamodel generator.

Reply:
Thank you for pointing to that!